packages/common/editors/__tests__/CheckboxEditor.spec.js

```diff
@@ -1,27 +1,45 @@
 const React          = require('react');
 const ReactDOM = require('react-dom');
 const TestUtils      = require('react-dom/test-utils');
-const CheckboxEditor = require('../CheckboxEditor');
 const { mount } = require('enzyme');
+const sinon = require('sinon');
+const CheckboxEditor = require('../CheckboxEditor');
 
 describe('CheckboxEditor', () => {
   let component;
   let componentWrapper;
+  let consoleErrorStub;
```

Comment:
I think check if console.error is called is not required. Can you revert this test?

Reply:
This test has been removed.